plugins/org.eclipse.xtend.ide.common/src/org/eclipse/xtend/ide/common/highlighting/XtendHighlightingCalculator.java

```diff
@@ -384,6 +384,7 @@ public void acceptSemanticLineBreak(int charCount, RichStringLiteral origin, boo
 		@Override
 		public void acceptTemplateLineBreak(int charCount, RichStringLiteral origin) {
 			resetCurrentOffset(origin);
+			acceptor.addPosition(currentOffset, charCount, TEMPLATE_LINE_BREAK);
```

Comment:
I can't follow the last remark: which method do you refer to?

TEMPLATE_LINE_BREAK seems to refer to line breaks that will be emitted to the output thus a semantic LB is a template linebreak in the highlighting sense (rather unfortunate that the highlighting key has such gotten this name).
TEMPLATE_LINE_BREAKs are LB that are not visible in the output and thus are not rendered at all. Thinking about it, I wonder why they should get a dedicated highlighting if the do not appear in the output. My gut feeling right now is that the change in the highlighter is not correct - it depends on the semantics that we want to have, though. If we stick to: text that appears in the output will get the template coloring, than the call to acceptor.acceptPosition(..) is wrong.